packages/calcite-components/src/components/carousel/interfaces.ts

```diff
@@ -1,4 +1,2 @@
-export type ArrowType = "inline" | "edge" | "none";
```

Comment:
We can leave component-specific types where they are. Anything that's shared should go in `src/components/interfaces.ts`. Some new types might need to be moved to component-level `interfaces.ts` files.